.devcontainer/setup.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash 
+set -euo pipefail
+
+# Go to the workspace directory: 
```

Comment:
Here's another comment. I don't know how to do this. This thing, I would ask AI testing for the presence of directory is not a good way to tell if you're in a container or not. So ask, you know, Chachi VT or AI or research for a better way to test if you're in the container. I don't know. Maybe this is the only way, but please review.